src/features/content-explorer/content-explorer/ContentExplorerSelectAll.js

```diff
@@ -0,0 +1,43 @@
+import PropTypes from 'prop-types';
+import React, { PureComponent } from 'react';
+import { injectIntl, FormattedMessage } from 'react-intl';
+import Checkbox from '../../../components/checkbox/Checkbox';
+
+import messages from '../messages';
+
+class ContentExplorerSelectAll extends PureComponent {
+    static propTypes = {
+        intl: PropTypes.any,
+        numTotalItems: PropTypes.number,
+        isSelectAllChecked: PropTypes.bool,
+        handleSelectAllClick: PropTypes.any,
+    };
+
+    static defaultProps = {
+        numTotalItems: 0,
+    };
+
+    render() {
+        const { intl, numTotalItems, handleSelectAllClick, isSelectAllChecked } = this.props;
+        const itemsCount = intl.formatNumber(numTotalItems);
+        return (
+            <div className="content-explorer-select-all-container">
+                <label className="content-explorer-select-all-items-counter">
```

Comment:
Can we use `Label` here?

Reply:
"Label" doesn't seam to be able to take className
so "label" is better for styling here